build.gradle.kts

```diff
@@ -30,9 +30,10 @@ val commonsTextVersion = "1.4"
 val arenaDialogNotatVersion = "1.e1999cf"
 val javaTimeAdapterVersion = "1.1.3"
 val vaultJavaDriveVersion = "3.1.0"
-val postgresVersion = "42.2.5"
+val postgresVersion = "42.2.13.jre6"
```

Comment:
Vi bruker vel ikke java-6?
Nyeste versjon er i hvert fall 42.3.1, ref:
https://search.maven.org/artifact/org.postgresql/postgresql/42.3.1/jar

Reply:
Jeg reagerte på det selv, men det var det Snyk mente jeg skulle gjøre. Endrer til 42.3.1 og ser hva som skjer :D

Reply:
Jepp. Hvis du ser på maven central så er det 3 varianter av hver versjon: 1 for java 6, 1 for java 7 og 1 "vanlig". Antar at den "vanlige" er for alle som er på java 8 og oppover, mens de to andre er custom-versjoner for de som bruker jdk/jre 6/7.